ExtraEnemyCustomization/Customizations/Models/SilhouetteCustom.cs

```diff
@@ -42,12 +42,60 @@ public void OnPrefabBuilt(EnemyAgent agent)
                 _MaterialCached = true;
             }
 
+            var renderers = agent.GetComponentsInChildren<Renderer>(true);
+            var rendererList = new List<Renderer>();
+            foreach (var renderer in renderers)
+            {
+                rendererList.Add(renderer);
+            }
+
+            var charMats = agent.GetComponentInChildren<CharacterMaterialHandler>().m_materialRefs;
+            foreach (var matRef in charMats)
+            {
+                if (!matRef.HasFeature(MaterialSupport.Destruction))
+                {
+                    RemoveFromRendererMatRef(matRef);
+                    continue;
+                }
+                if (!matRef.HasFeature(MaterialSupport.Destruction))
+                {
+                    RemoveFromRendererMatRef(matRef);
+                    continue;
+                }
+
+                foreach (var comp in matRef.m_renderers)
+                {
+                    if (comp.name.Equals("g_leg_l")) //MINOR: Shooter's left leg is always visible for some fucking reason
```

Comment:
This could be moved to some sort of readonly array that contains objects that should be ignored so it's easier to modify (?)